egs/mini_librispeech/s5/local/chain/tuning/run_tdnn_1j.sh

```diff
@@ -0,0 +1,300 @@
+#!/bin/bash
+
```

Comment:
Please don't have a separate link to this (RE `online_cmvn`.)-- I'd like to just advance the regular link. The idea is that it just becomes the new TDNN recipe.  (If we add a new suffix each time we change something it quickly becomes unmanageable).
The other task is to resolve conflicts.  Essentially this means moving forward j -> k and applying whatever changes I had in i->j.